newton/_src/utils/schema_resolver.py

```
@@ -0,0 +1,642 @@
+        # 3) Solver mapping defaults in priority order
+        for plugin in self.plugins:
+            specs = plugin.mapping.get(prim_type, {}).get(key, []) if hasattr(plugin, "mapping") else []
+            for spec in specs:
+                d = getattr(spec, "default", None)
+                if d is not None:
+                    return d
+
+        # Nothing found
+        try:
+            prim_path = str(prim.GetPath()) if prim is not None else "<None>"
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)